.github/workflows/generate-readmes.yml

```
@@ -4,6 +4,12 @@ on:
   push:
     branches:
       - changeset-release/develop
+  workflow_dispatch:
+    inputs:
+      BUILD_ALL:
+        description: true is certain steps should be run against all adapters or false if they should only run against some adapters
```

```suggestion
        description: whether certain steps should be run against all adapters or false if they should only run against some
```

---

Nitpick: I think your suggestion made sense for phrasing. Also fixes the typo
```suggestion
        description: whether certain steps should be run against all adapters
```